packages/web/config/generate-chain-infos/source-chain-infos.ts

```diff
@@ -4410,6 +4378,38 @@ export const mainnetChainInfos: SimplifiedChainInfo[] = [
     features: ["ibc-transfer", "ibc-go"],
     explorerUrlToTx: "https://vota-explorer.dorafactory.org/dora/tx/{txHash}",
   },
+  {
+    rpc: "", // TODO
+    rest: "", // TODO
```

Comment:
Should we follow up on this? @JeremyParish69

Reply:
Since Nomic's mainnet is not live, we don't have these values, but they should be available by the time #2317 is merged